src/components/Table/Table.stories.tsx

```diff
@@ -0,0 +1,966 @@
+        description: 'A CSS class to be applied to all cells in the column',
+      },
+      {
+        name: 'dataKey',
+        type: 'string',
+        description: 'The key in the rows object that matches this column',
+      },
+      {
+        name: 'emptyCellPlaceholder',
+        type: 'string | number | undefined',
+        description: 'placeholder for empty cells',
+      },
+      {
+        name: 'heading',
+        type: 'string',
+        description: 'The heading text to be displayed',
+      },
+      {
+        name: 'headerClassName',
+        type: 'string',
+        description: 'The CSS class to be applied to the column header cell.',
+      },
+      {
+        name: 'isSortable',
+        type: 'boolean',
+        description:
+          'Whether the column is sortable. Controls displaying the sort arrows and making the column header clickable',
+      },
+      {
+        name: 'key',
+        type: 'React.Key',
+        description:
+          'a custom key to be passed to each column. This gets autogenerated if not supplued',
+      },
+      {
+        name: 'render',
+        type: 'function: (cell, row, rowIndex) => ReactNode',
+        description:
+          'Render any custom content based on the cell and row content',
+      },
+      {
+        name: 'sticky',
+        type: "enum: 'left', 'right', ",
+        description:
+          'Whether the column will stick to the left or right side of the table when the table is larger than its parent',
+      },
+      {
+        name: 'truncateOverflow',
+        type: 'boolean',
+        description:
+          'Whether the column content will get truncated (with ellipsis) if it surpasses the fixed-width',
+      },
+      {
+        name: 'width',
+        type: 'number',
+        description: 'Specify the width of a particular column.',
+      },
+    ];
+    const columnConfig = [
+      { heading: 'Name', dataKey: 'name' },
+      {
+        heading: 'Type',
+        dataKey: 'type',
+        render: (cell: any) => <code style={codePreviewStyle}>{cell}</code>,
+      },
+      { heading: 'Description', dataKey: 'description' },
+    ];
+    return (
+      <Table rowKey="name" columns={columnConfig} rows={columnInterfaceRows} />
+    );
+  })();
+
+export const CommonExample = () =>
+  (() => {
+    const columnConfig = [
+      {
+        heading: 'Customer',
+        dataKey: 'customerInfo',
+        render: (_cell: any, row: any) => (
+          <div>
+            <div>{row.name}</div>
+            <div className="font-color-grey">{row.email}</div>
+          </div>
+        ),
+      },
+      { heading: 'phone', dataKey: 'phone', width: 125 },
+      { heading: 'plan', dataKey: 'plan', width: 125 },
+      { heading: 'status', dataKey: 'status', width: 125 },
+    ];
+    const tableData = [
+      {
+        status: 'lead captured',
+        name: 'John Smith',
+        email: 'john@palmetto.com',
```

Comment:
```suggestion
        email: 'john@hyphen.ai',
```
change all of these please